bin/onchange-nfs.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+if [[ `curl -s ${CONSUL}:8500/v1/health/state/passing | grep nfs`  ]]
+then
+  echo "nfs is healthy, mounting uploads directory...."
+  mount -t nfs -v -o nolock,vers=3 nfs:/exports /var/www/html/content/uploads
+  echo "removing no-uploads.php mu-plugin"
+  rm /var/www/html/content/mu-plugins/no-uploads.php
+  echo "adding 'upload_files' capability back to default roles"
+  # only these roles have 'upload_files' cap by default
+  for role in administrator editor author
+  do
+    if [ "$role" != 'role' ]
+    then
+      wp --allow-root cap add ${role} upload_files
+    fi
+  done
+else
+  echo "nfs is not healthly, umounting uploads directory..."
+  umount -f -l /var/www/html/content/uploads
+  echo "creating mu-plugin for NFS error in wp-admin"
+  echo "<?php
+/**
```

Comment:
This seems like a solid plan, but perhaps it makes more sense to keep it as a PHP file elsewhere and `cp` it into place as needed (then `rm` it when it's not)?